corehq/apps/hqwebapp/models.py

```diff
@@ -35,25 +35,29 @@ class MaintenanceAlert(models.Model):
 
     text = models.TextField()
     domains = ArrayField(models.CharField(max_length=126), null=True)
+    created_by_domain = models.CharField(max_length=255, null=True, db_index=True)
+    created_by_user = models.CharField(max_length=128, null=True)
 
     class Meta(object):
         app_label = 'hqwebapp'
+        db_table = 'hqwebapp_maintenancealert'
 
     @property
     def html(self):
         return mark_up_urls(self.text)
 
     def __repr__(self):
-        return "MaintenanceAlert(text='{}', active='{}', domains='{}')".format(
+        return "CommCareHQAlert(text='{}', active='{}', domains='{}')".format(
             self.text, self.active, ", ".join(self.domains) if self.domains else "All Domains")
 
     def save(self, *args, **kwargs):
-        MaintenanceAlert.get_active_alerts.clear(MaintenanceAlert)
-        super(MaintenanceAlert, self).save(*args, **kwargs)
+        CommCareHQAlert.get_active_alerts.clear(CommCareHQAlert)
+        super(CommCareHQAlert, self).save(*args, **kwargs)
```

Comment:
nit
```suggestion
        cls = type(self)
        cls.get_active_alerts.clear(cls)
        super().save(*args, **kwargs)
```

Reply:
Can you share why that is better?

Reply:
It doesn't require updating the code if the class name changes, which makes the code more resilient over time.

In the case of the `super()` call, it's been unnecessary to pass arguments since Python 3.0.